cockatrice/src/window_main.cpp

```diff
@@ -932,7 +932,8 @@ void MainWindow::createTrayIcon()
 void MainWindow::iconActivated(QSystemTrayIcon::ActivationReason reason)
 {
     if (reason == QSystemTrayIcon::DoubleClick) {
-        if (windowState() != Qt::WindowMinimized && windowState() != Qt::WindowMinimized + Qt::WindowMaximized)
+        if (windowState() != Qt::WindowMinimized &&
+            windowState() != ((int)Qt::WindowMinimized + (int)Qt::WindowMaximized))
```

Comment:
this makes no sense, the original writer of this line probably meant to use &

Reply:
Since the Qt ENUMS are key'd in 2 powers, adding and bit-wise ANDing should return the same value. It's not best practice, sure, but it does work. https://doc.qt.io/qt-5/qt.html#WindowState-enum

Reply:
adding and bitwise and do not do the same thing here, you're confused with bitwise or

what I mean is windowstate() & Qt::WindowMinimized == 0 replacing both these lines

Reply:
Meant to say OR, but yeah. I'll take a look at this